.werft/build.js

```diff
@@ -135,9 +135,20 @@ async function deployToDev(version) {
         werft.fail('secret', err);
     }
 
-    // TODO [geropl] Now that the certs reside in a separate namespaces, start the actual certificate issuing _before_ the namespace cleanup
-    werft.log('certificate', "organizing a certificate for the preview environment...");
-    const certificatePromise = issueAndInstallCertficate(namespace, domain);
+    werft.log("authProviders", "copy authProviders")
+    try {
+        exec(`kubectl get secret preview-envs-authproviders --namespace=keys --export -o yaml \
+                | yq r - data.authProviders \
+                | base64 -d -w 0 \
+                > authProviders`, {silent: true}).stdout.trim();
+        werft.done('authProviders');
```

Comment:
would it work to use `yq merge`  to merge the authProviders into the values.yaml?
This may allow us to avoid expanding the API surface of our helm chart.

Reply:
Good point!

Reply:
@meysholdt Switched to `yq merge`.